docs/index.txt

```diff
@@ -23,6 +23,7 @@ Are you new to Django or to programming? This is the place to start!
   :doc:`Part 5: Testing <intro/tutorial05>` |
   :doc:`Part 6: Static files <intro/tutorial06>` |
   :doc:`Part 7: Customizing the admin site <intro/tutorial07>`
```

Comment:
A "|" is missing on this line.

Reply:
Ah, 2 seconds — I'll adjust. Thanks @timgraham